### PR TITLE
determine if recipient identity change is unseen

### PR DESCRIPTION
**Pull request by michaelkirk**
PTAL @charlesmchen

### CODE REVIEW

File: src/Storage/AxolotlStore/TSStorageManager+IdentityKeyStore.m [hunk 1]
```diff
@@ -199,6 +199,31 @@ - (nullable OWSRecipientIdentity *)unconfirmedIdentityThatShouldBlockSendingForR
     }
 }
 
+- (nullable OWSRecipientIdentity *)unseenIdentityChangeForRecipientId:(NSString *)recipientId
+{
+    OWSAssert(recipientId != nil);
+
+    @synchronized([[self class] sharedIdentityKeyLock])
+    {
+        OWSRecipientIdentity *currentIdentity = [OWSRecipientIdentity fetchObjectWithUniqueID:recipientId];
```
Comment (charlesmchen) [inline src/Storage/AxolotlStore/TSStorageManager+IdentityKeyStore.m:208]: Should we use `sessionDBConnection` for identities?

**Review by charlesmchen — APPROVED** [review]
LGTM

File: src/Storage/AxolotlStore/TSStorageManager+IdentityKeyStore.m (hunk 1 above)
Comment (michaelkirk) [reply src/Storage/AxolotlStore/TSStorageManager+IdentityKeyStore.m:208]: Hmmm... why? Keep in mind that OWSRecipientIdentity already have a similarly constrained (no-cache) dbConnection.

Comment (charlesmchen) [reply src/Storage/AxolotlStore/TSStorageManager+IdentityKeyStore.m:208]: Should we combine these two connections?  NBD.

**Pull request merged**